pages/querying/functions.mdx

```diff
@@ -157,12 +157,70 @@ This section contains the list of built-in query modules and their procedures.
 
 ### Schema
 
-The `schema` procedures return information about the database schema.
+The `schema` procedures return information about the database schema and can be used to assert constraints and indices exist in the database.
```

Comment:
```suggestion
The `schema` procedures return information about the database schema and can be used to assert constraints and indices exist in the database.
```

What do you mean by assert?

Reply:
It is a bit stupid term I know but Neo4j uses that one. Assert that constraint and indices exist in the database.